skll/utilities/print_model_weights.py

```diff
@@ -58,6 +64,13 @@ def main(argv=None):
     learner = Learner.from_file(args.model_file)
     (weights, intercept) = learner.model_params
 
+    multiclass = False
+    model = learner._model
+    if (isinstance(model, LinearSVC) or
+        isinstance(model, LogisticRegression) or
+        (isinstance(model, SVC) and
+            model.kernel == 'linear')):
+        multiclass = True
```

Comment:
You could be using `LogisticRegression` for bianary classification in which case you will only have weights for one class and then you don't need to do this. May be you need an condition that the number of classes (via `self.label_list`) is  > 2?